siliconcompiler/toolscripts/ubuntu24/install-openroad.sh

```
@@ -13,31 +13,30 @@ fi
 
-deps_args=""
-if [ ! -z ${PREFIX} ]; then
-    deps_args="-prefix=$PREFIX"
-fi
-sudo ./etc/DependencyInstaller.sh -base
-$SUDO_INSTALL ./etc/DependencyInstaller.sh -common $deps_args
+sudo ./etc/DependencyInstaller.sh -bazel -prefix="$BAZEL_PREFIX"
+sudo chown -R $USER:$USER $BAZEL_PREFIX
 
-cmake_args="-DENABLE_TESTS=OFF"
 if [ ! -z ${PREFIX} ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)